monitoring/statistics.cc

```diff
@@ -121,6 +121,18 @@ void StatisticsImpl::recordTick(uint32_t tickerType, uint64_t count) {
   }
 }
 
+void StatisticsImpl::reduceTick(uint32_t tickerType, uint64_t count) {
+  assert(
+    enable_internal_stats_ ?
+      tickerType < INTERNAL_TICKER_ENUM_MAX :
+      tickerType < TICKER_ENUM_MAX);
+  per_core_stats_.Access()->tickers_[tickerType].fetch_sub(
```

Comment:
assert the return value not smaller than `count`?

Reply:
Good idea.